index.bs

```diff
@@ -86,7 +86,7 @@ figure.table figcaption {
     counter-increment: none;
 }
 figure.table figcaption:not(.no-marker)::before {
-    content: "Table " counter(table) " ";
+    content: counter(table);
```

Comment:
Why strip the `Table ` portion? It looks a little odd to have these naked table numbers.
![image](https://user-images.githubusercontent.com/518542/67426408-e0f7ad00-f58e-11e9-995a-799da644a0cb.png)

Reply:
(I think the "Table" might need to be put back in manually otherwise you can't search for it and nor can one choose the capitalisation at will.)

Reply:
D'oh. Good catch, thanks!

Reply:
Yes, searchability was a main motivation for this, but I accidentally removed the prefix from the `<figcaption>` rule as well. Probably in part because there's no corresponding rule for figures (`<figure>`s without `.table`), I guess `<figcaption>`s have a default "Figure N" prefix built in. So both before and after fixing this, the prefix in the `<figcaption>` isn't searchable, but the references are.